uint8 MISI_1 = 1
uint8 MISI_2 = 2
uint8 MISI_3 = 3
uint8 FINISH = 4